### DISCUSSION TITLE
Combining phases from ebsd

### DISCUSSION THREAD

**Discussion by JasonOttUW** [mtex-toolbox/mtex#1273] [Ask Anything]
In an EBSD map, I have 2 garnet phases indexed: almandine and pyrope, but would like to treat both phases as just a generic garnet for the analysis I am doing. Is there an easy way that I can concatenate these two phases together?

**Comment by kilir** [comment]
Hi,
here's an example of one possibility:
```
mtexdata olivine
ebsd
% assign phase 4 -> 3
ebsd('Chalcopyrite').phase = 3
```
Alternatively, you can alter ebsd.CSList in case you'd like to introduce something new.
cheers,
Rüdiger

**Comment by JasonOttUW** [comment]
Hello, Rudiger,

Thanks for your response! I decided to try reimporting the EBSD file into
MTEX setting the CS info for both almandine and pyrope to match, and it
appears to have worked-I just did this and I see that the orientations in
the garnet phase match the sun of the 2 separate phases on my previous
work. I was working with the file looking for any issues I may have
caused--can you forsee any problems with this approach?

Thanks for your help,

Jason
